### ISSUE TITLE
progress bar into the blu screen for multiple enable/disable?

### ISSUE THREAD

**Issue by 3D-I** [javiexin/extdepend#5]
![multiple_ext_enable-disable_1](https://cloud.githubusercontent.com/assets/480857/21911039/f9a58f6e-d91e-11e6-9d4e-ebd14171c278.png)
So to take more attention and don't leave that screen before the job it's done.

**Comment by javiexin** [comment]
I'm all for this... Please, propose some visual, and I will take a look.  Preferably without javascript.

**Comment by 3D-I** [comment]
I do have something already, from a mod of mine (for 3.0.x). That's HTML though.
I used it for an installer. I will be on this soon.

Instead, Synphony have something for progress bars, I read it and of it's application on the core code (maybe the CLI installer), if I correctly recall.

**Comment by 3D-I** [comment]
To add, found the Synphony's progress bar used there
https://github.com/phpbb/phpbb/blob/b7061af193465f60dca1fdeafadbe7f544ed50ea/phpBB/phpbb/install/helper/iohandler/cli_iohandler.php

It looks appropriate for such message screen, I saw it in action somewhere.

**Comment by javiexin** [comment]
From what I can see, this is a text-only bar, I believe.  I think that if we are to include this, it should be somehow graphical... maybe similar to the way poll results are presented, to keep some level of consistency.
Anyhow, I have not seen this progress bar in use myself, so probably the first thing would be to see it in action.

**Comment by 3D-I** [comment]
Yes, in that case. But's customizable, In fact it's used also in the installer (see installer/installer.php of 3.2.0) and widely used in the core here and there.

Here's the Symphony's Doc for the progress bar built-in
http://symfony.com/doc/current/components/console/helpers/progressbar.html

Another source for a gfx'd progress bar could be the progress bar of the attachments.

**Comment by 3D-I** [comment]
what about a CSS loader?
https://projects.lukehaas.me/css-loaders/
I saw this merged for 3.3.0

https://github.com/lukehaas/css-loaders

**Comment by javiexin** [comment]
Hi Marco,  I will look into it.  My priority now is improving functionality in other extensions more than cosmetic changes, but I will look at it.
For the record: I am working on making extension breakages not crash the whole forum (will improve the proposed core PR, and will make it to the Improved Ext Mgr), and will incorporate Composer Installer, Dependency Mgr and Event lists to Improved Ext Mgr as well...
And a few other things, like Scheduled Posts, migrating Advanced Polls to 3.2...
Too much on my plate!

**Comment by javiexin** [comment]
Have looked at the CSS loader.  Really, they are more of a "working" indicator than a progress bar.
I think that, for this use, a real progress bar would work better.  Maybe with some animation to give the feedback that it is working...  The real issue is that for each operation, the cache is purged, and everything must be initialized again (container, extensions, config, permissions... everything).

**Comment by 3D-I** [comment]
Yes. I see..
Take your time, is not so important against other things that you are after.
Me too I am quite busy last times, say buried under a pile of code.
Thanks mate.
